apiary.apib

```diff
@@ -1196,6 +1251,13 @@ A business asset type group. Used to classify a business asset type and apply ba
 + `value`: General Maintenance (string, required) - The description of the purpose.
 + `isActive`: true (boolean, required) - Indicates if the purpose is active.
 
+## Trade Create (Base Audit Object)
```

Comment:
The trade create does not need to extend the base audit object because those fields are readonly